### PR TITLE
[SecurityBundle] Added an alias from RoleHierarchyInterface to security.role_hierarchy

### PR DESCRIPTION
**Pull request by lyrixx**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | I don't know
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets |
| License       | MIT
| Doc PR        |

It's needed for autowirering

---

Note: I'm not sure if this is a bug fix or a new feature.

### CODE REVIEW

File: src/Symfony/Bundle/SecurityBundle/Resources/config/security.xml [hunk 1]
```diff
@@ -96,6 +96,7 @@
         <service id="security.role_hierarchy" class="Symfony\Component\Security\Core\Role\RoleHierarchy">
             <argument>%security.role_hierarchy.roles%</argument>
         </service>
+        <service id="Symfony\Component\Security\Core\Role\RoleHierarchy" alias="security.role_hierarchy" />
```
Comment (chalasr) [inline src/Symfony/Bundle/SecurityBundle/Resources/config/security.xml]: Should be `RoleHierarchyInterface` to me

Comment (lyrixx) [reply src/Symfony/Bundle/SecurityBundle/Resources/config/security.xml]: Arf, I noticed that too, but it looks like I forgot to push. Thanks.

**Review by nicolas-grekas — APPROVED** [review]

**Review by xabbuh — APPROVED** [review]

**Review by chalasr — APPROVED** [review]

**Comment by chalasr** [comment]
Thank you @lyrixx.

**Comment by buffcode** [comment]
Will this be backported to 3.4?

**Comment by lyrixx** [comment]
@buffcode No, it's a new feature and all new features go to master.
Buy you can easily add this alias in **your** `services.yml` ;)

**Comment by buffcode** [comment]
Thanks for clarification, yeah I know it is easy to add.
I'm just adding this to any 3.4 project as of now, so it's some kind of PITA ;)

**Pull request merged**